data/categories.txt

```diff
@@ -7511,7 +7518,7 @@ pl:Park
 pt:Parque|parque urbano
 pt-BR:Parque|praça
 ro:Parc
-es:Parque
+es:Parque|Plaza
```

Comment:
Please remove "Plaza". The user should be able to search for a square or a park separately.